src/canopy_bioemi_mod.F90

```diff
@@ -143,6 +165,12 @@ SUBROUTINE CANOPY_BIO( ZK, FCLAI, FCH, LAI, FSUN, PPFD_SUN, &
             GAMMACO2 = 1.0_rk
         end if
 
+! Get LEAF AGE factor
+        TABOVECANOPY  = TLEAF_AVE  !Assume instantaneous TLEAF estimate for TABOVECANOPY
+        do i=1, SIZE(ZK)
+            GAMMALEAFAGE(i) = GET_GAMMA_LEAFAGE(LEAFAGEOPT, PLAI, CLAI, NDAYS, TABOVECANOPY(i), ANEW, AGRO, AMAT, AOLD)
```

Comment:
Thus, since TABOVECANOPY is just a 1D/2D variable from input model/obs (based on 2-m temperature), seems to me that GAMMALEAFAGE should only be a scalar here, and also not indexed on model layers (ZK).